src/Model/ModelManager.php

```diff
@@ -761,10 +761,19 @@ public function modelTransform($class, $instance)
         return $instance;
     }
 
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in version 4.0.
```

Comment:
```suggestion
     * @deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in version 4.0.
```
```suggestion
     * @deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in version 4.0. Use "reverseTransform()" instead.
```